bin/dev

```diff
@@ -1,3 +1,9 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
Could this be `sh` instead because we might want to run this command inside `docker` which uses Alpine and doesn't come with bash preinstalled?

The `if` below might need to change :thinking: